src/Controls/src/Core/VisualElement.cs

```diff
@@ -203,7 +203,12 @@ static void OnTransformChanged(BindableObject bindable, object oldValue, object
 
 		public static readonly BindableProperty OpacityProperty = BindableProperty.Create("Opacity", typeof(double), typeof(VisualElement), 1d, coerceValue: (bindable, value) => ((double)value).Clamp(0, 1));
 
-		public static readonly BindableProperty BackgroundColorProperty = BindableProperty.Create("BackgroundColor", typeof(Color), typeof(VisualElement), null);
+		public static readonly BindableProperty BackgroundColorProperty = BindableProperty.Create(nameof(BackgroundColor), typeof(Color), typeof(VisualElement), null,
+			propertyChanged: (bindable, oldvalue, newvalue) =>
+			{
+				if (bindable is VisualElement element)
+					element.Handler?.UpdateValue(nameof(Background));
```

Comment:
Should this get tied into via the mappers? 

we can use the new RemapToControls stuff to add an entry for "BackgroundColor" and then have that call this UpdateValue call